### PR TITLE
Remove ComponentStatus from external_services (jaeger, grafana, prom)

### PR DESCRIPTION
**Pull request by xeviknal**
Requires https://github.com/kiali/kiali/pull/3423

It remove the ComponentStatus from jaeger, prom and grafana. It leaves only one param for users tune the severity of the status in the masthead.

### CODE REVIEW

**Review by hhovsepy — APPROVED** [review]

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
Looks like this is removing some settings that the `null-cr-values-test` molecule test is setting.

See: https://github.com/kiali/kiali-operator/blob/master/molecule/null-cr-values-test/kiali-cr.yaml#L72 (and other places). Just make sure that test CR is still valid.

File: roles/default/kiali-deploy/defaults/main.yml [hunk 1]
```diff
@@ -152,10 +149,6 @@ kiali_defaults:
       cache_duration: 7
       cache_enabled: true
       cache_expiration: 300
-      component_status:
-        app_label: "prometheus"
-        is_core: true
-        namespace: ""
```
Comment (jmazzitelli) [inline roles/default/kiali-deploy/defaults/main.yml:158]: Does Prometheus have a "core_component" field also? If so, it should be specified here and documented in the `kiali_cr.yaml` so users know about it.

**Comment by xeviknal** [comment]
I've updated the molecule test for the nulls replacing the previous flag by the new one.
thank you @hhovsepy and @jmazzitelli.

**Review by jmazzitelli — DISMISSED** [review]
LGTM

File: deploy/kiali/kiali_cr.yaml [hunk 2]
```diff
@@ -419,10 +419,7 @@ spec:
 #   use_kiali_token: When true and if auth.type is "bearer", the same OAuth token used for authentication in Kiali will be used for the API calls to Grafana,
 #       and auth.token config is ignored then.
 #   username: Username to be used when making requests to Grafana, for basic authentication. User only requires viewer permissions.
-# component_status: Grafana info for the Istio Component Status masthead indicator. Only if it is enabled.
-#   app_label: The value of the Grafana pod(s)'s app label. The name of the app label is defined in the setting 'istio_labels.app_label_name'. It defaults to grafana.
-#   is_core: Whether the component is core for your deployment. It defaults to false.
-#   namespace: The namespace where Grafana is installed in. It defaults to the 'istio_namespace' setting.
+# core_component: Used in the Components health feature. When true, the unhealthy scenarios will be raised as errors. Otherwise, they will be raised as a warning.
```
Comment (jotak) [inline deploy/kiali/kiali_cr.yaml:422]: I think I prefer with the `is_` prefix to make it clear that a boolean is expected without having to refer to doc

**Comment by xeviknal** [comment]
@jotak @hhovsepy approached this one too. thx

**Review by jotak — APPROVED** [review]
LGTM

**Pull request merged**